azext_iot/common/utility.py

```diff
@@ -146,6 +150,44 @@ def process_json_arg(content: str, argument_name: str = "content", preserve_orde
         )
 
 
+def process_yaml_arg(path: str) -> dict:
```

Comment:
nit, might as well use a Dict generic for return type like in the process toml function